services/director-v2/tests/unit/with_dbs/test_route_clusters.py

```diff
@@ -96,13 +101,19 @@ async def test_list_clusters(
     response = await async_client.get(list_clusters_url)
     assert response.status_code == status.HTTP_200_OK
     returned_clusters_list = parse_obj_as(List[ClusterGet], response.json())
-    assert len(returned_clusters_list) == 111
+    assert len(returned_clusters_list) == 111 + 1
```

Comment:
not sure what this 111 + 1 is all about. Maybe have some constants to make it more readable?

Reply:
did you actually read the test?